src/jdk.compiler/share/classes/com/sun/source/util/DocTreeFactory.java

```diff
@@ -413,6 +413,19 @@ default SummaryTree newSummaryTree(List<? extends DocTree> summary) {
      */
     ValueTree newValueTree(ReferenceTree ref);
 
+    /**
+     * Creates a new {@code ValueTree} object, to represent a {@code {@value }} tag.
+     * @param format a format string for the value
+     * @param ref a reference to the value
+     * @return a {@code ValueTree} object
+     *
+     * @implSpec This implementation calls {@link #newValueTree(ReferenceTree) newValueTree(ref)}.
+     * @since 19
```

Comment:
ditto

Reply:
Those `@since` tags were correct when the PR was approved, but became outdated on the 9th of June. What can I say? A PR has shelf life.

Those tags should be fixed. @jonathan-gibbons, please file a bug and I will review it. @ExE-Boss, thanks for noticing.